src/components/common/CreatorCard.tsx

```diff
@@ -124,7 +124,41 @@ const CreatorCard: React.VFC<{
   noPadding,
 }) => {
   const { formatMessage } = useIntl()
-  const { enabledModules } = useApp()
+  const { enabledModules, settings } = useApp()
+
+  // optional condition
+  const tabs = settings['creator.content.tab']?.split(',') || []
+
+  // module -> tabs ? tabs : sys default -> none
+  const actions = [
+    {
+      condition: tabs.length !== 0 ? (tabs.includes('program') ? withProgram : false) : withProgram,
```

Comment:
could define `tabs.length !== 0` as `const hasTabs = tabs.length !== 0`

Reply:
As described above, this condition can be simplified to `hasTabs ? tabs.includes('program') : withProgram`

Reply:
Since the includes check is equivalent to checking whether tabs has a value, it can be further simplified.
`tabs.includes('program') ?? withProgram`

Reply:
I apologize for my comment above
`tabs.length !== 0` shold be change to `Boolean(tabs)`, an empty array is also a assigned value